app/views/overview/_service-instance-row.html

```diff
@@ -14,13 +14,18 @@
           </div>
         </h3>
         <div class="status-icons">
+          <!--
+            TODO: this should show an info status icon if the apiObject is not yet ready.
+          -->
           <notification-icon ng-if="!row.expanded" alerts="row.notifications"></notification-icon>
+          <span ng-if="!row.isReady">
+            <status-icon status="row.status"></status-icon> {{row.statusReason | sentenceCase}}
```

Comment:
I'm opting for #2, the less verbose. Objections @jwforres @spadgett ?
![screen shot 2017-04-12 at 2 25 38 pm](https://cloud.githubusercontent.com/assets/280512/24973378/7ca0835c-1f8c-11e7-9cac-e77fde3a3468.png)
![screen shot 2017-04-12 at 2 29 34 pm](https://cloud.githubusercontent.com/assets/280512/24973377/7c9fb9e0-1f8c-11e7-9a40-54686ed39271.png)